docs/ref/settings.txt

```diff
@@ -2644,6 +2644,13 @@ A ready-to-use instance of the storage backends can be retrieved from
 :data:`django.core.files.storage.storages`. Use a key corresponding to the
 backend definition in :setting:`STORAGES`.
 
+.. note::
```

Comment:
To improve readability, use `.. admonition:: Descriptive title` rather than `.. note::` as described in [Guidelines for reStructuredText files](https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#guidelines-for-restructuredtext-files).